src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+const personStatus = (sex) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return partner;
+};
+
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName } = person;
```

Comment:
if you have more than 3 props, more correctly place it in column, it will be more cleaner
```suggestion
  const { 
    name, 
    age, 
    sex, 
    partnerName 
  } = person;
```